pykern/pkcli/fmt.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+u"""Wrapper for Python formatter (currently, ``black``) to update and to validate a repository.
+
+:copyright: Copyright (c) 2015 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkdebug import pkdp
+import pykern.pksubprocess
+import pykern.pkunit
+
+def run(path):
+    '''
+        Run black formatter on path
+
+        Args:
+            path - string path to file or dir
+    '''
+    pykern.pksubprocess.check_call_with_signals(['black', f'{path}'])
+
+
+def diff(path):
+    pykern.pksubprocess.check_call_with_signals([
+        'black',
+        '--diff',
+        '--no-color',
+        f'{path}'
+    ])
+
+
+
+def check(path):
+    '''Returns True if there would be diff else return False'''
+    try:
+        pykern.pksubprocess.check_call_with_signals([
```

Comment:
rule of three says to factor out `pykern.pksubprocess.check_call_with_signals(["black"` as it is repeated.